functions/slack/app/format.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018, Google, LLC.
```

Comment:
No comma - try running `license_test.go` at the root of the repo (`go test -v .` at the repo root). Make sure to merge the latest changes from master because I updated `badfiles_test.go`.

Reply:
Done